app/_helpers/fake-backend.ts

```diff
@@ -1,154 +1,156 @@
 ﻿import { Http, BaseRequestOptions, Response, ResponseOptions, RequestMethod, XHRBackend, RequestOptions } from '@angular/http';
 import { MockBackend, MockConnection } from '@angular/http/testing';
 
-export let fakeBackendProvider = {
-    // use fake backend in place of Http service for backend-less development
-    provide: Http,
-    useFactory: (backend: MockBackend, options: BaseRequestOptions, realBackend: XHRBackend) => {
-        // array in local storage for registered users
-        let users: any[] = JSON.parse(localStorage.getItem('users')) || [];
-
-        // configure fake backend
-        backend.connections.subscribe((connection: MockConnection) => {
-            // wrap in timeout to simulate server api call
-            setTimeout(() => {
-
-                // authenticate
-                if (connection.request.url.endsWith('/api/authenticate') && connection.request.method === RequestMethod.Post) {
-                    // get parameters from post request
-                    let params = JSON.parse(connection.request.getBody());
-
-                    // find if any user matches login credentials
-                    let filteredUsers = users.filter(user => {
-                        return user.username === params.username && user.password === params.password;
-                    });
-
-                    if (filteredUsers.length) {
-                        // if login details are valid return 200 OK with user details and fake jwt token
-                        let user = filteredUsers[0];
-                        connection.mockRespond(new Response(new ResponseOptions({
-                            status: 200,
-                            body: {
-                                id: user.id,
-                                username: user.username,
-                                firstName: user.firstName,
-                                lastName: user.lastName,
-                                token: 'fake-jwt-token'
-                            }
-                        })));
-                    } else {
-                        // else return 400 bad request
-                        connection.mockError(new Error('Username or password is incorrect'));
-                    }
+function fakeBackendFactory(backend: MockBackend, options: BaseRequestOptions, realBackend: XHRBackend) {
```

Comment:
ERROR in Error encountered resolving symbol values statically. Reference to a non-exported function (position 4:10 in the original .ts file), resolving symbol fakeBackendProvider in ...